book/chapter-02.md

```diff
@@ -1,16 +1,29 @@
+
+Consequently, to ensure the safety of the Republic, the Founding Fathers sought to decentralize power and allow each and every individual in the country to own the most powerful weapon available at the time – a really crappy gun. This is what the Founding Fathers loved about it. They were a perfect decentralization of power. Moving forward a couple hundred years, we see many blockchains falling under this same spell. Satoshi wanted to design a system where the power to control money and information was as widely distributed as possible – a very, very big deal. 
+
+And just as the Founding Fathers turned to the gun, Satoshi turned to the CPU. Just like the guns of the 1790’s, CPUs are enormously powerful, yet widely distributed among the public, and few individuals have more than one or two. This means that this awesome power (the CPU), as Satoshi imagined, was reasonably distributed among the masses and controlled by no central entity. And for the first couple of years, Bitcoin was a virtual paradise of decentralization. Indeed, each year it became more and more decentralized. 
+But it is dangerous to decentralize based on technology, because technology changes. And so today, we might still call an AR-16 a gun, but from the perspective of the Founding Fathers, that would be like calling an armor plated cheetah a house cat. There wasn’t a single person alive in the late 1700’s, who, when they embraced the striking beauty of the 2nd amendment, considered something as deadly as modern weaponry. Returning to today, we see that blockchains like Bitcoin and others who rely on client-side validation through the use of miners have their own automatic rifle problem. It is called an ASIC. And an ASIC is unlike a CPU in that it is entirely useless to most folks (just as an AR-16 is today). As such, because few folks own them, ASICS are the opposite of decentralization. 
+
+Indeed, just as a few folks bought CPUs to mine, every purchase of an ASIC has been with the intention of rigging the system for the individual. The Founding Fathers and Satoshi both imagined a world where tech would create decentralization. But because they used tech, the opposite has happened. While we cross our fingers that one day, everyone will have an equal power to mine cryptocurrency. It does not seem likely. 
+
+The Founding Fathers were philosophers, not gun toters. And like Satoshi, they were chasing a dream of decentralization. But they both seemed to make the same mistake – they relied on technology. Decentralization, however, will always rely on people. 
```

Comment:
I saw disagree. Trust relies on people and thus unreliable in itself. That is why we turn to technology where we instead put our trust in mathematics and cryptography as opposed to human trust.

Reply:
Good point - couldn't agree more. I seemed to veer away from the whole basis cryptoeconomics lies on. Where traditional econonomics relies on people making rational decisions that will provide the most optimal level of benefit or utility - cryptoeconomics realizes that is a pretty big assumption and not reliable. But it was the best we had at the time. 

Where we used to rely on rational people making rational decisions every time,  Cryptoeconomics uses mathematics and cryptographic proofs as opposed to human trust. Creating a neutral system humans can actually trust. 

Will update later tonight after work, thanks Peter.